src/js/components/Box.js

```diff
@@ -3,11 +3,20 @@
 import React, { Component, PropTypes } from 'react';
 import KeyboardAccelerators from '../utils/KeyboardAccelerators';
 import Intl from '../utils/Intl';
+import Props from '../utils/Props';
 import SkipLinkAnchor from './SkipLinkAnchor';
 
 const CLASS_ROOT = "box";
 
 export default class Box extends Component {
+  constructor(props) {
+    super(props);
+    this.restProps = Props.omit(props, Object.keys(Box.propTypes));
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.restProps = Props.omit(nextProps, Object.keys(Box.propTypes));
```

Comment:
@ericsoderberghp Do you think there should be some props change checks here or this is ok?